Sources/StreamChatUI/CommonViews/AvatarView/ChatUserAvatarView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// A view that shows a user avatar including an indicator of the user presence (online/offline).
+public typealias ChatUserAvatarView = _ChatUserAvatarView<NoExtraData>
+
+/// A view that shows a user avatar including an indicator of the user presence (online/offline).
+open class _ChatUserAvatarView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
+
+    /// A view that shows the avatar image and online presence indicator.
+    open private(set) lazy var onlineAvatarView: _ChatOnlineAvatarView<ExtraData> = uiConfig
+        .onlineAvatarView.init()
+        .withoutAutoresizingMaskConstraints
+
+    /// The data this view component shows.
+    open var content: _ChatUser<ExtraData.User>? {
+        didSet { updateContentIfNeeded() }
+    }
+
+    override open func setUpLayout() {
+        super.setUpLayout()
+        embed(onlineAvatarView)
+    }
+
+    override open func updateContent() {
+        guard let user = content else { return }
```

Comment:
I don't think we need this check. We need to reset the UI when the content is `nil`

Reply:
Yeah good point 👍

Reply:
✅